src/App.jsx

```diff
@@ -1,33 +1,15 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from 'react';
+import './App.css';
+import ArticlesList from './components/ArticlesList';
+
 
 function App() {
   const [count, setCount] = useState(0)
```

Comment:
You don't appear to be doing anything with this state at the moment? It can therefore be removed.